src/rashmi/BankingSystem.java

```diff
@@ -0,0 +1,99 @@
+/*Assignment 9 : 2nd April 2021
+
+Create a Banking System which has the following functionality.
+
+Create two object to perform below scenario.
+1) Method to debit amount
+2) Method to credit amount
+3) Method to printBalance [it will print current balance]
+4) individualTransactionSummary() method should show how many times individually debit, credit & printBalance method called .
+Hint : Create 3 non static instance variable.
+Output :
+user1 transaction summary : Credit - 2 times, Debit - 1 times, printBalance - 1 time
+user2 transaction summary : Credit - 5 times, Debit - 2 times, printBalance - 0 time
+
+5) allTransactionSummary() method should show by both users total how many times debit, credit & printBalance method get called.
+Hint : Create 3 static variables will be required.
+Output :
+All transaction summary : Credit - 7 times, Debit - 3 times, printBalance - 1 time*/
+
+
+
+package rashmi;
+public class BankingSystem{
+
+	int balance;
+	int debitcnt=0;
+	int creditcnt=0;
+	String name;
+	static int  totalDebitCnt=0;
+	static int totalCreditCnt=0;
+	int printBalancecnt = 0;
+	static int totalprintBalancecnt = 0;
+	
+	 
+
+	void setData(String uname, int initialBalance){
+		name = uname;
+		balance = initialBalance;
+	}
+	
+	void debtAmt(int amt){
+		while(balance>=amt){
```

Comment:
this should be if condition, think on this

Reply:
yes right..changes done..Thank you